shell_integration/nautilus/setappname.sh

```diff
@@ -3,4 +3,4 @@
 # this script replaces the line
 #  appname = 'ownCloud'
 # with the correct branding name in the syncstate.py script
-sed -i.org -e 's/appname\s*=\s*'"'"'ownCloud'"'/appname = '$1'/" syncstate.py
+sed -i.org -e "s/ownCloud/$1/g" syncstate.py
```

Comment:
The comment is now outdated. Also, minor, I think replacing a different identifier "APP_NAME" or something) would make problems with this replace less likely.